src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+import java.util.Scanner;
+//import java.util.Arrays;
+
 public class Duke {
+
+    private static final Scanner input = new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput = "";
+        Task[] currentTasks = new Task[100];
```

Comment:
Perhaps you can refractor the number 100 to show the meaning of it since its a constant?